sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -288,15 +288,20 @@ def get_all_options(self,
       _LOGGER.warning("Discarding unparseable args: %s", unknown_args)
     result = vars(known_args)
 
+    overrides = self._all_options.copy()
     # Apply the overrides if any
     for k in list(result):
+      overrides.pop(k, None)
       if k in self._all_options:
         result[k] = self._all_options[k]
       if (drop_default and
           parser.get_default(k) == result[k] and
           not isinstance(parser.get_default(k), ValueProvider)):
         del result[k]
 
+    if  overrides:
```

Comment:
Looks like there are two spaces.

Reply:
Good catch. I don't know why lint did not catch it.